src/httpd_jsonapi.c

```diff
@@ -2634,6 +2634,111 @@ jsonapi_reply_library_album_tracks(struct httpd_request *hreq)
   return HTTP_OK;
 }
 
+static int
+jsonapi_reply_library_tracks_get_byid(struct httpd_request *hreq)
+{
+  time_t db_update;
+  struct query_params query_params;
+  const char *track_id;
+  struct db_media_file_info dbmfi;
+  json_object *reply = NULL;
+  int ret = 0;
+
+  db_update = (time_t) db_admin_getint64(DB_ADMIN_DB_UPDATE);
+  if (db_update && httpd_request_not_modified_since(hreq->req, &db_update))
+    return HTTP_NOTMODIFIED;
+
+
```

Comment:
Double new line?

Reply:
I did that in the other functions too. It is to add a separation between the not modified check and the rest. Maybe not the best way to do it, but i would like to leave it like that for now. 
I was thinking about generalizing the check for not modified (it is identical in most cases), but that would better be done in a separate pr.